media-gfx/blender/blender-9999.ebuild

```diff
@@ -168,10 +169,18 @@ pkg_setup() {
 src_unpack() {
 	if [[ ${PV} = *9999* ]] ; then
 		git-r3_src_unpack
+
+		EGIT_REPO_URI=${ADDONS_EGIT_REPO_URI}
+		EGIT_CHECKOUT_DIR=${S}/scripts/addons
+
+		git-r3_src_unpack
```

Comment:
Don't override the vars, this will confuse the hell out of smart-live-rebuild.

Instead leave the main repo in vars and call `git-r3_fetch` and `git-r3_checkout` with the appropriate arguments directly for other repos.